assignment1.html

```diff
@@ -0,0 +1,51 @@
+<html>
+    <head>
+        <title>My Guest Book</title>
+        <link rel="stylesheet" type="text/css" href="styles.css">
+    </head>
+    <body>
+    <div id="h">My Guest Book</div>
+    <div id="c"> Please sign my guest book.Thanks!</div><br><br>
+    
+    <img  id="img1" src="D:\kannan\cat.jpg">
+    <img id="img2" src="D:\kannan\suburb.jpg">
```

Comment:
It is not good to use absolute file paths in HTML files, as these may not work on other devices. You copy the images to a same location as HTML files (or add them to a images folder).
```html
<img  id="img1" src="./cat.jpg">
<img id="img2" src="./images/suburb.jpg">
```